evap/settings.py

```diff
@@ -418,6 +419,10 @@
     # give random char field values a reasonable length
     BAKER_CUSTOM_FIELDS_GEN = {"django.db.models.CharField": lambda: random_gen.gen_string(20)}
 
+    # csrf tokens are added in JS and do not work in the tests at all. The django test runner
+    # does disable CSRF protection by default, but the WebTest app does not. So we disable it here.
+    MIDDLEWARE = [m for m in MIDDLEWARE if m != "django.middleware.csrf.CsrfViewMiddleware"]
+
```

Comment:
A small defense against the unindent-attack that @He3lixxx mentioned could be to insert a `assert TESTING` here, so that indentation is broken if only the middleware line is unindented but the assertion fails if both are

Another thought: Will malicious whitespace-only diffs still tell us if a line was joined with the (commented) line above?

Reply:
good idea. looks a bit odd but I mean why not.

I'm pretty sure joining lines does not count as whitespace-change.